src/sentry/integrations/slack/event_endpoint.py

```diff
@@ -93,6 +93,50 @@ def _parse_url(self, link):
     def on_url_verification(self, request, data):
         return self.respond({"challenge": data["challenge"]})
 
+    def on_message(self, request, integration, token, data):
+        channel = data["channel"]
+        # if it's a message posted by our bot, we don't want to respond since
+        # that will cause an infinite loop of messages
+        if data.get("bot_id"):
+            return self.respond()
+
+        access_token = integration.metadata.get("user_access_token")
```

Comment:
@MeredithAnya Maybe we should make a utility function to get the access token from the metadata since we use this in `on_link_shared`